app/controllers/declaration/SummaryController.scala

```diff
@@ -64,23 +69,41 @@ class SummaryController @Inject()(
   }
 
   val submitDeclaration: Action[AnyContent] = (authenticate andThen verifyEmail andThen journeyType).async { implicit request =>
-    LegalDeclaration.form.bindFromRequest
-      .fold(
+    def submit(form: Form[LegalDeclaration]): Future[Result] =
+      form.bindFromRequest.fold(
```

Comment:
This might be by design but as the form binding is happening here, if the LRN is a duplicate then no form errors from the legal declaration form will be detected or shown to the user. Generally we try to display all the errors on the page at the same time but maybe this error is special? (Maybe check with Liz and/or Johnathan)?

Reply:
That was an oversight on my part, I have now added the LRN duplication error alongside so it will appear with any others.